pkg/server/sandbox_portforward_unix.go

```diff
@@ -100,12 +100,12 @@ func (c *criService) portForward(ctx context.Context, id string, port int32, str
 			return ctx.Err()
 		}
 		// give a chance to terminate gracefully or timeout
-		// 0.5s is the default timeout used in socat
+		// after 1s
 		// https://linux.die.net/man/1/socat
-		timeout := time.Duration(500) * time.Millisecond
+		const timeout = time.Second
 		select {
 		case e := <-errCh:
-			if errFwd == nil {
+			if errFwd == nil && e != nil {
```

Comment:
I think this can result in a nil assignment, which would make the check in L122 fail.

Reply:
it only will be a nil assignment if errFwd was nil. Otherwise, errFwd keeps the error obtained from the first io.Copy() independently of the value of `e`

Reply:
Yes, removed that additional check